google-datacatalog-kafka-connector/README.md

```diff
@@ -126,17 +126,47 @@ export KAFKA2DC_KAFKA_HOST=kafka_bootstrap_server
 
 ```
 
+Export below variables to enable optional ingestion of the schemas from the Schema Registry. 
+The url is required, the rest depends on your Schema Registry authentication setup.
+
+```bash
+# Required to connect to the Schema Registry
+
+export KAFKA2DC_SCHEMA_REGISTRY_URL=url/to/schema/registry
+
+# All the following are optional and depend on your Schema Registry authentication setup
```

Comment:
It would be useful to add a link to the authentication options (since you mention it here), and instruct users if they have other needs to open a feature request.

Reply:
Done